Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -264,8 +264,14 @@ def check_for_latest_xquartz
 
         <<-EOS.undent
           Your XQuartz (#{installed_version}) is outdated.
-          Please install XQuartz #{latest_version} (or delete it):
-            https://xquartz.macosforge.org
+          Please install XQuartz #{latest_version} (or delete the current version).
+          XQuartz can be installed via homebrew-cask with
+
+            brew cask install xquartz
+
+          or manually, by downloading the package from
+
+            https://www.xquartz.org/
```

Comment:
Ok! I copied this format generally from e.g. https://github.com/Homebrew/brew/blob/19347f3b87917368d7157fe5007d506a4db0c7db/Library/Homebrew/cask/lib/hbc/exceptions.rb#L87 - should that be changed as well?